salt/modules/cp.py

```diff
@@ -900,7 +903,7 @@ def push_dir(path, glob=None, upload_path=None):
         return push(path, upload_path=upload_path)
     else:
         filelist = []
-        for root, _, files in os.walk(path):
+        for root, _, files in os.walk(ur''.join(path)):
```

Comment:
Yup I already mentioned this in the issue forum. we have to use `r''`  instead of `ur''`. Does we stick with python3 or is the code should be compatible with both python2 and 3 ?